sync-tube.py

```diff
@@ -142,10 +142,14 @@ def youtube_dl_download(url):
 # processes - how many processes to use
 #
 def download_videos_pool(videos, processes, verbose):
+    global YOUTUBE_DL_OPTIONS
+
     YOUTUBE_DL_OPTIONS['progress_hooks'] = [youtube_dl_hook]
     YOUTUBE_DL_OPTIONS['logger'] = YoutubeDLLogger(verbose)
+
     with Pool(processes=processes) as pool:
-        pool.map(youtube_dl_download, videos)
+        #Note: we have to pass YOUTUBE_DL_OPTIONS as an arg, because for some reason Windows doesn't see changes in the global var??
+        [pool.apply(youtube_dl_download, args = (url, YOUTUBE_DL_OPTIONS)) for url in videos]
```

Comment:
I think adding  `YOUTUBE_DL_OPTIONS` as a default parameter for `youtube_dl_download` would look prettier than using the `global` keyword

Reply:
Hopefully this will still work on Windows

Reply:
Do you mean like this?
def youtube_dl_download(url, options = YOUTUBE_DL_OPTIONS):

Reply:
That's what I meant but really isn't a solution.

Can we try doing something like
```python
def youtube_dl_download_with_options(options):
    def youtube_dl_download(video):
        code using options
    return youtube_dl_download
```

And then using `pool.map` again?

Reply:
I have commited a fix already, can you check if it is working properly too?

Reply:
Thank you so much for your contribution, I,m planning to make this software more user friendly and generally better so stay tuned bc there will be more issues soon

Reply:
> I have commited a fix already, can you check if it is working properly too?

Works as expected in Windows and Ubuntu.

Since your have came up with a better fix, imma close the pull request.

Also, if you continue working on this, i'd recommend tagging working releases, so people don't happen on the repo, and pull a broken commit lol